000-debug/debug-mode.php

```diff
@@ -25,8 +25,8 @@
 add_action( 'muplugins_loaded', __NAMESPACE__ . '\init_debug_mode' );
 
 function init_debug_mode() {
-	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-	$enable = filter_var( $_GET['a8c-debug'] ?? '', FILTER_SANITIZE_STRING );
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	$enable = $_GET['a8c-debug'] ?? '';
```

Comment:
Could `sanitize_text_field()` be used here?

Reply:
@GaryJones yes, but we use `$enable` only to compare its value against two predefined constants, that is why I decided not to waste CPU cycles.

Reply:
Makes sense.